client/src/sections/quests/Level4.tsx

```diff
@@ -0,0 +1,53 @@
+import Line from "../../components/Line";
+import QuestTile from "../../components/QuestTile";
+import { useObjectFunction } from "../../hooks/useFunction";
+import { atStep, Step, useQuestStep } from "../../hooks/useQuest";
+
+const Level4: React.FC = () => {
+    const step = useQuestStep("questL04Bat");
+    const bodyguards: { turnsSpent?: number } = useObjectFunction.toLocation(
+        "The Boss Bat's Lair"
+    );
+
+    if (step === Step.FINISHED) return <Line></Line>;
```

Comment:
```suggestion
    if (step === Step.FINISHED) return <></>;
```
you want it to actual show nothing here

Reply:
also needs tab changes